testing/tests/DevExpress.serverSide/themes.tests.js

```diff
@@ -1,59 +1,17 @@
 const themes = require('ui/themes');
-const viewportUtils = require('core/utils/view_port');
 
-window.includeThemesLinks();
-themes.setDefaultTimeout(20);
+QUnit.module('themes');
 
-const viewport = document.createElement('div');
-viewport.className = 'dx-viewport';
-document.body.appendChild(viewport);
-
-QUnit.module('themes', {
-    beforeEach: function() {
-        this._originalTheme = themes.current();
-    },
-    afterEach: function() {
-        viewportUtils.value(viewport);
-        themes.current(this._originalTheme);
-    }
-});
-
-QUnit.test('theme changing', function(assert) {
-    const done = assert.async();
-    themes.current('generic.light');
-
-    assert.equal(themes.current(), 'generic.light');
-
-    themes.initialized(() => {
```

Comment:
The reason why these tests were deleted is not clear

Reply:
They were wrong because it is not possible to change theme on the server side and there are not theme links on the server side.